core/api/src/services/twilio-service.ts

```diff
@@ -57,6 +58,9 @@ export const TwilioClient = (): IPhoneProviderService => {
         fields: "line_type_intelligence",
       })
       // https://www.twilio.com/docs/lookup/v2-api/line-type-intelligence#type-property-values
+      if (!lookup.lineTypeIntelligence) {
```

Comment:
do we really want to stop sending the code if `lineTypeIntelligence`  is not defined?

Reply:
https://github.com/GaloyMoney/galoy/pull/3758#discussion_r1437226749